betterdiscordctl

```diff
@@ -188,29 +192,35 @@ while :; do
   shift
 done
 
+# currently unused
+mkdir -p "$data"
```

Comment:
Let's not create a directory we're not going to use.

Reply:
Whoops, meant to comment out that line.